src/run.js

```diff
@@ -302,7 +302,7 @@ const processPage = ({
 /**
  *
  * @param {{ urls: Array<string>, debug: boolean, loadimages: boolean, skippable: function, browser: any, userAgent: string, withoutjavascript: boolean, viewport: any }} options
- * @return Promise<{ finalCss: string, stylesheetContents: string }>
+ * @return Promise<{ finalCss: string, stylesheetContents: Object.<string, string> }>
```

Comment:
To be honest this unfamiliar syntax to me. I will need to read about it

Reply:
I just skimmed http://usejsdoc.org/tags-type.html
It says "An object with string keys and number values:" for `{Object.<string, number>}`

Reply:
The problem is that TypeScript JSDoc is not the same as "vanilla" JSDoc. [There are tone of JSDoc flavors](https://github.com/stereobooster/type-o-rama#jsdoc-flavours). This is what supported by TypeScript https://github.com/Microsoft/TypeScript/wiki/JSDoc-support-in-JavaScript

Reply:
oh really. But isn't this better than `string`? :)

Reply:
I suppose this is the right syntax for it

```typescript
Promise<{ finalCss: string, stylesheetContents: { [key: string]: string } }>
```

but typescript doesn't care (I mean `yarn tsc`)

Reply:
PR updated.